mm/alignment.py

```diff
@@ -391,6 +391,97 @@ def hash(cls, kwargs):
         )
 
 
+class PlotAlignmentJob(Job):
+    """
+    Plots an alignment from its log file.
+    This is a separate job from the tasks available in :func:`i6_core.mm.alignment.AlignmentJob.plot`
+    and :func:`i6_core.
+    """
+
+    def __init__(
+        self,
+        alignment_log_file: tk.Path,
+        clip_low: Optional[int] = None,
+        clip_high: Optional[int] = None,
+        clip_percentile_low: Optional[int] = None,
+        clip_percentile_high: Optional[int] = None,
+        num_bins: int = 50
+    ):
+        """
+        :param alignment_log_file: Alignment log file from the alignment job.
+        :param clip_low: Number symbolizing the absolute number at which the plot will be clipped to the left.
+            Can't be given in conjunction to :param:`clip_percentile_low`. By default clips at the minimum value.
+        :param clip_high: Number symbolizing the absolute number at which the plot will be clipped to the right.
+            Can't be given in conjunction to :param:`clip_percentile_high`.
+            By default clips at the 95th percentile (for removing huge outliers).
+        :param clip_percentile_low: Number symbolizing the percentile at which the plot will be clipped to the left.
+            Can't be given in conjunction to :param:`clip_low`. By default clips at the minimum value.
+        :param clip_percentile_high: Number symbolizing the absolute number at which the plot will be clipped to the right.
+            Can't be given in conjunction to :param:`clip_high`.
+            By default clips at the 95th percentile (for removing huge outliers).
+        :param num_bins: Number of histogram bins. By default `50`.
+        """
+        self.alignment_log_file = alignment_log_file
+        clip_low_dict = {"clip_low": clip_low, "clip_percentile_low": clip_percentile_low}
+        clip_low_diff_none_values = [lower_val for lower_val in clip_low_dict.values() if lower_val is not None]
+        assert len(clip_low_diff_none_values) <= 1, (
+            f"More than one value for the low clip explicitly given (different from None): {clip_low_dict.items()}. "
+            "Please only give one value."
+        )
```

Comment:
That sounds like `clip_low is None != clip_percentile_low is None` in very complicated.

But should we even disallow both? we could just take whatever is lower/higher (i mean more restrictive) and go with it.

Reply:
Done, thanks for the suggestion.